linux/host_verify_saml_token/host_verify_saml_token.yml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This test case is used for test host verified SAML token in guest operations
+#
+- name: host_verify_saml_token
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: "Test case block"
+      block:
+        - name: "Skip test case due to missing vCenter Server variables"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: "Skip test case {{ ansible_play_name }} is because of missing vCenter Server variables."
+            skip_reason: "Not Applicable"
+          when: >
+            (vcenter_is_defined is undefined or not vcenter_is_defined) or
+            (vcenter_ssh_username is undefined or not vcenter_ssh_username) or
```

Comment:
Where are these 2 parameters 'vcenter_ssh_username' and 'vcenter_ssh_password' been used?

Reply:
`common/add_vsphere_hosts_in_inventory.yml` added vcenter_hostname to inventory, and `common/vcenter_manage_domain_user.yml` will use it for adding VC domain user